zimui/package.json

```diff
@@ -42,8 +44,9 @@
     "eslint-plugin-vue": "^9.23.0",
     "jsdom": "^24.0.0",
     "npm-run-all2": "^6.1.2",
-    "ogv": "^1.9.0",
+    "ogv": "1.8.9",
```

Comment:
Please add a comment with a link to PR comment explaining why we pinned this version.

Reply:
Comments are not allowed in JSON files. Do you have any suggestions where else I can mention this?

Reply:
Damn it ; let's live without it then